stratum/hal/lib/tdi/tdi_id_mapper.cc

```diff
@@ -65,6 +65,11 @@ ::util::Status TdiIdMapper::PushForwardingPipelineConfig(
           p4extern.extern_type_id() != kTnaExternActionSelectorId) {
         continue;
       }
+      if (p4extern.extern_type_id() == kTnaExternActionSelectorId) {
+        // DirectTSMeter (extern id 132) externs must be skipped because they
```

Comment:
The comments: It would be a nice to explicitly to put the field "extern_instance.preamble field is not present in tdi.json" instead of "they" ?